install.sh

```diff
@@ -397,4 +396,4 @@ nvm_reset() {
 
 [ "_$NVM_ENV" = "_testing" ] || nvm_do_install
 
-} # this ensures the entire script is downloaded #
+} # this ensures the entire script is downloaded #:
```

Comment:
The colon doesn’t do anything inside a comment, is there a reason this needs to be here?

Reply:
yeah probably from exiting vim.